cwltool/sandboxjs.py

```diff
@@ -1,12 +1,11 @@
-import cStringIO
```

Comment:
pycharm finds a lot, but that is mostly things like `typing.List` and `typing.Union`, which are probably imported by default by mypy.